storage/bootstrap.go

```diff
@@ -193,7 +206,7 @@ func (m *bootstrapManager) bootstrap() error {
 	// all the data we bootstrapped.
 	rateLimitOpts := m.fsManager.RateLimitOptions()
 	m.fsManager.SetRateLimitOptions(rateLimitOpts.SetLimitEnabled(false))
-	m.fsManager.Run(m.nowFn(), false)
+	m.fsManager.Run(m.nowFn(), false, true)
```

Comment:
Makes sense. Took me a second to get that we need it to indicate that we should flush despite the db not being bootstrapped.